yoti-sdk-api/src/main/java/com/yoti/api/client/identity/constraint/ConstraintConstants.java

```diff
@@ -0,0 +1,7 @@
+package com.yoti.api.client.identity.constraint;
+
+public class ConstraintConstants {
+
+    public static final String SOURCE_CONSTRAINT = "SOURCE";
+
+}
```

Comment:
I don't get this one... I though you moved the value within the `SourceConstraint` class... why have it declared here too?